packages/core/admin/admin/src/core/utils/axiosInstance.js

```diff
@@ -1,6 +1,10 @@
 import axios from 'axios';
 import { auth } from '@strapi/helper-plugin';
 
+console.log(
```

Comment:
I'm not sure putting at the root of the modules is great as it will log on import so even before the functions are used. 

We could create a simple util to wrap the axiosInstance to log a warning on each call instead ?

Reply:
Yes, I agree. A central place for this warning would be nice, because for the start I'd like it be wrapped in a NODE_ENV=development check. As soon as we have good progress on the migration we can remove that but for now I'd like to avoid so many warnings at the same time.

Reply:
So a wrapper you could build could be very simple like 

```js

const wrapAxiosDeprecationWarnings = (axiosInstance) => {


return {
	get(...arguments) {
		console.log('warning');
		return axiosInstance.get(...arguments)
	},
	......
  }


// or automated based on the doc here https://axios-http.com/docs/instance (might not be the matching version)

const wrapper = {};

['request', 'get', 'head', 'delete'', 'options', 'post', 'put', 'patch', 'getUri' ].forEach(methodName => {

wrapper[methodName] = (...args) => axiosInstance[methodName]
})

return wrapper
}

```

Reply:
so basically I have to replace every call to the axiosInstance and instead call the wrapAxiosDeprecationWarnings?

Reply:
No, you will need to export wrapAxiosDeprecationWarnings(axiosIsntance) from the axiosInstance files only

Reply:
ok thanks